lib/vdsm/storage/lvmdevices.py

```diff
@@ -78,6 +79,10 @@ def configure(vgs):
             _configure_devices_file(enable=False)
         raise
 
+    # We are done with configuration of lvm devices file and lvm filter is not
+    # needed/used any more. Remove it, if there is any.
+    lvmfilter.remove_filter()
```

Comment:
We have to remove this before configuring to solve the issue with filter= warning.

Reply:
actually it can be done in a way that this is not needed